docs/docs/modules.json

```diff
@@ -55,6 +55,7 @@
         "glm",
         "OpenXR",
         "libusb",
+        "libuvc",
```

Comment:
usb video devices.  I added it because it was required to build under Ubuntu 24, and there is no harm in including it. Though we don't require it directly, some of the packages we use require it, so it is added for completeness.